src/Buckle/Buckle.Tests/CodeAnalysis/EvaluatorTests.cs

```diff
@@ -646,17 +650,26 @@ expected identifier
     private void AssertDiagnostics(string text, string diagnosticText, bool assertWarnings = false) {
         var annotatedText = AnnotatedText.Parse(text);
         var syntaxTree = SyntaxTree.Parse(annotatedText.text);
-        var compilation = Compilation.CreateScript(null, syntaxTree);
-        var result = compilation.Evaluate(new Dictionary<VariableSymbol, object>());
```

Comment:
This is a bug fix to the testing framework, it wasn't stopping compilation if the parser had errors when it should have been